h2integrate/converters/water/desal/desalination.py

```diff
@@ -126,11 +126,17 @@ def setup(self):
             merge_shared_inputs(self.options["tech_config"]["model_inputs"], "cost")
         )
 
-    def compute(self, inputs, outputs):
-        """Cost reference: https://www.nrel.gov/docs/fy16osti/66073.pdf"""
+    def compute(self, inputs, outputs, discrete_inputs, discrete_outputs):
+        """Cost reference: Table 3 of https://www.nrel.gov/docs/fy16osti/66073.pdf.
+        CapEx includes 2.55% financing factor, numbers based on INL report
+        https://doi.org/10.2172/1236837 (in Table 10) which came from this report
+        https://www-pub.iaea.org/MTCD/Publications/PDF/te_1561_web.pdf
+        (Table 6 says 2006 is the currency reference year)
+        """
         desal_capex = 32894 * (self.config.freshwater_kg_per_hour / 3600)  # [USD]
 
         desal_opex = 4841 * (self.config.freshwater_kg_per_hour / 3600)  # [USD/yr]
 
         outputs["CapEx"] = desal_capex
         outputs["OpEx"] = desal_opex
+        discrete_outputs["cost_year"] = 2013  # TODO: check this value
```

Comment:
I realize I already asked you to check this year due to the docs page. Thanks for noting that it's a TODO already!